integration/common/testutils.go

```diff
@@ -0,0 +1,139 @@
+package common
```

Comment:
Per https://github.com/golang/go/wiki/CodeReviewComments#package-names, avoid package names like `common`.

I suggest `integrationtest` as package name, and `integrationtest.Run` as function name.